react/components/loaders/LoadingSpinner.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 import { Spinner } from 'vtex.styleguide'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = [
+  'loadingSpinnerOuterContainer',
+  'loadingSpinnerInnerContainer',
+]
 
 const LoadingSpinner = props => {
   const { loading } = props
+  const handles = useCssHandles(CSS_HANDLES)
   return (
     loading && (
-      <div className="w-100 flex justify-center">
-        <div className="w3 ma0">
+      <div
+        className={`${handles.loadingSpinnerOuterContainer} w-100 flex justify-center`}
+      >
+        <div className={`${handles.loadingSpinnerOuterContainer} w3 ma0`}>
```

Comment:
Ops should be inner here
```suggestion
        <div className={`${handles.loadingSpinnerInnerContainer} w3 ma0`}>
```